OpenRA.Mods.Common/Traits/Render/WithSpriteTurret.cs

```diff
@@ -25,6 +25,12 @@ public class WithSpriteTurretInfo : ConditionalTraitInfo, IRenderActorPreviewSpr
 		[Desc("Sequence name to use")]
 		[SequenceReference] public readonly string Sequence = "turret";
 
+		[Desc("Custom palette name")]
+		[PaletteReference] public readonly string Palette = null;
+
+		[Desc("Custom PlayerColorPalette: BaseName")]
+		[PaletteReference(true)] public readonly string PlayerPalette = null;
```

Comment:
I think we just expose a `IsPlayerPalette` boolean elsewhere. Any specific reason you deviated from this here?

Reply:
This is probably copied from RenderSprites tbh.

Reply:
Yup it is.

Reply:
Yeah, and that makes sense on RenderSprites - not over here though.

Reply:
I agree that this should be consistent with
https://github.com/OpenRA/OpenRA/blob/7d59602c0ba3150a6151f44d0af195925a96f872/OpenRA.Mods.Common/Traits/Render/WithIdleOverlay.cs#L32-L36

@IceReaper can you please change this?